awx/ui/client/lib/components/list/_index.less

```diff
@@ -25,6 +25,28 @@
     margin-bottom: @at-margin-bottom-list-toolbar;
 }
 
+.at-List-toolbar--attached + .at-List{
+    .at-List-container {
+        .at-Row--active{
+            border-radius:0 0 0 0;
+        }
+    }
+}
+
+.at-List-toolbar--attached + .at-List{
+    .at-List-container > div:last-of-type {
+        border-bottom-left-radius:@at-border-radius;
+    }
+}
+
+.at-List-container > div:first-of-type{
+    border-top-left-radius:@at-border-radius;
```

Comment:
I don't think we need these styles. The active border is getting styled here: https://github.com/ansible/awx/blob/a76d22facc913edcc49d5fc7b49abc55b3fc2d3b/awx/ui/client/lib/components/list/_index.less#L152